controllers/api.controllers.js

```diff
@@ -0,0 +1,10 @@
+const selectApi = require('../models/api.models.js')
+
+function getApi(req, res, next){
+
+    selectApi().then((result) =>{
+        res.status(200).send(result)
```

Comment:
It's good practice to send back your responses on a descriptive key - so in a situation like this it would be better to send something like `{endpoints: *endpoints data*}` back so that it's more clear what that response actually is